json2args/parameter.py

```diff
@@ -61,6 +61,24 @@ def _parse_param(key: str, val: str, param_config: dict):
             with open(val, 'r') as f:
                 val = json.load(f)
         return val
+    elif t.lower() in ('integer', 'float'):
+        # check for min and max values in config
+        min = c.get('min', None)
+        max = c.get('max', None)
+
+        # check wether val is in min and max range
+        if min and max:
+            # check if min is smaller than max
+            if max < min:
```

Comment:
maybe even `<=` or is `min==max` an accepted edge-case?

Reply:
Yes, I think `<=` is better, because `min==max` would mean that only one number is allowed, and then it would not really make sense to specify it as a parameter, since the value is constant.